appcircle_ios_test_report/1.0.4/component.yaml

```diff
@@ -0,0 +1,41 @@
+platform: iOS
+buildPlatform: ObjectiveCSwift, Flutter
+displayName: Test Reports
+description: Parse test and coverage results.
+docsUrl: https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests#generating-test-report
+requiredComponents: "appcircle_ios_test"
+precedingComponents: "appcircle_ios_xcresult_converter_html_xml"
+followingComponents: "appcircle_export_build_artifacts"
+webUrl: https://github.com/appcircleio/appcircle-test-report-component
+repoUrl: https://github.com/appcircleio/appcircle-test-report-component.git
+commit: 02cbb2a
```

Comment:
If the PR on the component side has been merged, can you please get the latest commit from the repo for the PREP update?